packages/dx-react-chart/docs/guides/getting-started.md

```diff
@@ -4,7 +4,7 @@ Project status: **Beta**
 
 ## Overview
 
-The DevExtreme React Chart is a component that visualizes data using a variety of series types, including bar, line, area, scatter, pie, and more. It is a stateless component - it relies on properties and works in controlled mode only. The DevExtreme React Chart has a composable and extendable architecture in which plugins provide additional elements (such as axes, legend, grid). Twitter Bootstrap and Material-UI rendering and theming are supported out of the box.
+The DevExtreme React Chart is a component for data visualization that provides many series types, including bar, line, area, scatter, pie, and more. It can function in uncontrolled and controlled state modes. In uncontrolled mode, UI plugins manage the state internally. In controlled mode, the state is managed externally via plugin props. The DevExtreme React Chart has a composable and extendable architecture in which plugins provide additional elements (such as axes, legend, grid). Twitter Bootstrap and Material-UI rendering and theming are supported out of the box.
```

Comment:
"The DevExtreme React Chart is a component for data visualization that provides many series types, including bar, line, area, scatter, pie, and more." -> The DevExtreme React Chart is a data visualization component that provides different series types, including bar, line, area, scatter, pie, and so on.

Reply:
In **the** uncontrolled mode, **the** UI plugins manage the state internally.

Reply:
(such as a legend, grid, and axes)

Reply:
It also supports Twitter Bootstrap and Material-UI rendering and theming.